server/commands.go

```diff
@@ -56,74 +56,75 @@ func NewServerCommands(defaultCfg *sconfig.Config) []*cli.Command {
 					Name:    common.FlagDBPath,
 					Aliases: []string{"f"},
 					Value:   defaultCfg.DatabaseFilePath,
-					Usage:   "File in which to persist Temporal state",
+					Usage:   "File in which to persist Temporal state (by default, Workflows are lost when the process dies)",
 				},
 				&cli.StringSliceFlag{
 					Name:    common.FlagNamespace,
 					Aliases: common.FlagNamespaceAlias,
-					Usage:   "Specify namespaces that should be pre-created. Namespace 'default' is auto created",
+					Usage:   `Specify namespaces that should be pre-created (namespace "default" is always created)`,
 					EnvVars: nil,
 					Value:   nil,
 				},
 				&cli.IntFlag{
 					Name:    common.FlagPort,
 					Aliases: []string{"p"},
-					Usage:   "Port for the temporal-frontend GRPC service",
+					Usage:   "Port for the frontend gRPC service",
 					Value:   sconfig.DefaultFrontendPort,
 				},
 				&cli.IntFlag{
 					Name:  common.FlagMetricsPort,
-					Usage: "Port for the metrics listener",
+					Usage: "Port for /metrics",
 					Value: sconfig.DefaultMetricsPort,
+					DefaultText: "disabled",
 				},
 				&cli.IntFlag{
 					Name:        common.FlagUIPort,
-					Usage:       "port for the temporal web UI",
+					Usage:       "Port for the Web UI",
 					DefaultText: fmt.Sprintf("--port + 1000, eg. %d", sconfig.DefaultFrontendPort+1000),
 				},
 				&cli.BoolFlag{
 					Name:  common.FlagHeadless,
-					Usage: "disable the temporal web UI",
+					Usage: "Disable the Web UI",
 				},
 				&cli.StringFlag{
 					Name:    common.FlagIP,
-					Usage:   `IPv4 address to bind the frontend service to instead of localhost`,
+					Usage:   `IPv4 address to bind the frontend service to`,
 					EnvVars: nil,
 					Value:   "127.0.0.1",
 				},
 				&cli.StringFlag{
 					Name:        common.FlagUIIP,
-					Usage:       `IPv4 address to bind the web UI to instead of localhost`,
-					DefaultText: "same as --ip (eg. 127.0.0.1)",
+					Usage:       `IPv4 address to bind the Web UI to`,
+					DefaultText: "same as --ip",
 				},
 				&cli.StringFlag{
 					Name:    common.FlagLogFormat,
-					Usage:   `customize the log formatting (allowed: ["json" "pretty"])`,
+					Usage:   `Set the log formatting. Options: ["json", "pretty"].`,
```

Comment:
as another option, no strong opinion. So this in another cli and liked
```suggestion
					Usage:   `Set the log formatting. Supports: 'json', 'pretty'`,
```

Reply:
I like this better. I didn't change the array format because I saw that elsewhere in the code, it's printed with a Go stringification command. Can those be changed to this format as well?

Reply:
ok i'm good with the current approach. If we decide to change the format we can do this separately so it's consistent. Ps saw the `Supports 'xxx', 'yyy'` format in kubectl and liked it because it is short and clean

Reply:
👍 to be clear, by "this" i meant your suggestion 😊